### ISSUE TITLE
odoo.com/forum page stop responding and cause 100% chrome CPU, after I remove the <header> element

### ISSUE THREAD

**Issue by odoochina-dev** [odoo/owl#831]
Steps to reproduce is stated here: 
[https://github.com/odoo/odoo/issues/65151](https://github.com/odoo/odoo/issues/65151)

Delete DOM element is not normal operation from odoo/odoo pov, thus closed there.  But I think there must be some kind of bug (dead loop), could you please kindly comments from owl framework pov?

> No, I believe remove element from DOM or add new element to DOM is quite normal, that's why jQuery have .remove(), .detach(), .append(), ... etc., this kind of APIs.
And what ever kind of operation, it shall not cause 100% CPU... IMHO.

**Comment by ged-odoo** [comment]
Hmm, after investigation, this is unrelated to Owl. Owl does not render the forum page.
